### ISSUE TITLE
Timeseries prediction questions

### ISSUE THREAD

**Issue by Richterr** [Microsoft/CNTK#1158]
Hello,
I am learning timeseries prediction using the [existing tutorial](https://github.com/Microsoft/CNTK/blob/master/Tutorials/CNTK_104_Finance_Timeseries_Basic_with_Pandas_Numpy.ipynb), and I have a couple of questions:

- Shouldn't array with predictions be shifted forwards by `input_dimensions-2`? Currently it gives a prediction   right from the first element of the `test_data`, but in my mind it's supposed to be zero for the first `input_dimensions-2` elements, since it uses them for actual prediction. Please correct me if I'm wrong.  

- Regarding the tutorial: why was non-recurrent network used?  

- Why `learning_rate` and `minibatch_size` values were locked at `0.125` and `100`?  

- Not a question: it says that we can't use `Adj. Close` in the tutorial, but it actually uses it twice, I think that's a typo, and `Close` was used the first time

Sorry if my questions seem dumb, I'm only learning. Thank you.

**Comment by sayanpa** [comment]
Predictions: This is a tutorial showing how to do timeseries predictions. Please experiment with the data / model and study their impact.

Recurrent network based tutorial: Handling of time series using Recurrent networks is currently in the works and should be available in couple of weeks.

Learning rate and minibatch_size: The learning rate is something empirically determined. You can try something else. The same thing with minibatch _size.

Adj close: is used in the test data as reference and not used for modeling.

**Comment by Richterr** [comment]
> Recurrent network based tutorial: Handling of time series using Recurrent networks is currently in the works and should be available in couple of weeks.  

Relly looking forward for this one! Thanks for the answers

**Comment by mg64ve** [comment]
Hi, any update on this?
Is the tutorial is available?

**Comment by sayanpa** [comment]
Here they are:
https://github.com/Microsoft/CNTK/blob/master/Tutorials/CNTK_106A_LSTM_Timeseries_with_Simulated_Data.ipynb (in the master)

Another one in code review:
https://github.com/Microsoft/CNTK/blob/guschmue/tut-106b/Tutorials/CNTK_106B_LSTM_Timeseries_with_IOT_Data.ipynb

We have another tutorial on density forecasting in the works.

**Comment by mg64ve** [comment]
Interesting tutorials, thanks for sharing.
However, these tutorials work on 1D time series.
I wonder to know if there is any documentation on LSTM with multiple features.
Do you have an idea where to look for?
I have tried to search in this site but I could not find anything.

**Comment by eldakms** [comment]
Closing due to inactivity, there was no new tutorial on this if i'm not mistaken, Sayan can provide more info. Feel free to reopen if this is still relevant. Thanks!

**Comment by sayanpa** [comment]
Please look at tutorial CNTK 202 where a feature vector is provided as input. You can do the same here.
